build.gradle

```diff
@@ -1,11 +1,14 @@
 plugins {
-    id "com.vanniktech.maven.publish" version "0.19.0" apply false
+    id "com.vanniktech.maven.publish" version "0.22.0"
```

Comment:
from v0.23.0 requires JDK 11 or later, so fix v0.22.0 for now
https://github.com/vanniktech/gradle-maven-publish-plugin/blob/main/CHANGELOG.md#0230-2022-12-29